src/admin/units/units.component.tsx

```diff
@@ -48,13 +49,15 @@ function Units() {
         header: 'Value',
         accessorFn: (row) => row.value,
         id: 'value',
+        filterFn: 'fuzzy',
         Cell: ({ row }) => row.original.value,
       },
       {
         header: 'Last modified',
-        accessorFn: (row) => new Date(row.modified_time),
+        accessorFn: (row) => removeSecondsFromDate(row.modified_time),
```

Comment:
```suggestion
        accessorFn: (row) => removeSecondsFromDate(row.modified_time),
```

why is the removed seconds from date needed ??

Reply:
Previously, dates would be stored to the second, but would only be displayed and filtered to the minute. This creates some unexpected behaviour for users. For instance a value of `01/01/24 13:23 05 seconds` would be excluded with an upper bound of '01/01/24 13:23', since the extra seconds put it out of range. Therefore, I created and added the function to all of the date values in the columns.

Reply:
That's not unexpected behavior when using a date range, The user should just extend the range to  `01/01/24 13:24` to include all possible values of  `01/01/24 13:23:00 -  01/01/24 13:23:59`.

Reply:
You're correct in that it's not unexpected behaviour for a date range working with date storing seconds. However, the user only sees dates displayed to the minute (as shown below), so they wouldn't know dates are stored to the second, and therefore would not think to extend the range by a minute to capture all possible values.
![image](https://github.com/user-attachments/assets/739540a7-7bbc-4802-a52c-2b1212751973)
Removing the seconds from the data in the accessor functions shouldn't harm the user experience in anyway, since they can only filter down to the minute.

Reply:
This is not a issue for use to solve. The user should use the data range correctly. If they wanted items at 10:07 the range should be 10:06 to 10:08.

Reply:
Admittedly I am biased here as I suggested it as an issue 😄.  But I agree with Anikesh’s point – a user deciding what to filter by are given a false impression of the behaviour which I believe is our problem to solve. However as we spoke yesterday about this, there another flaw in that the minimum field would really need to be taken as 0 seconds, and the maximum as 59 seconds for the between filters to work like this rather than just setting the seconds to 0 (e.g. a between filter of 10:07 and 10:08 would exclude 10:07:30 - it would only effect between inclusive), so I think it would be best a separate task anyway if we have between inclusive included.

Reply:
I agree that it probably should be left for a seperate task. I spoke to Jousha about this earlier today, and suggested making a request/leaving a comment to MRT for them to fix the issue. 

I will remove the code from this PR. In regards to raising the issue with MRT's creators, do I just make a bug report on their repo, or do something else?

Reply:
Yes you would do that, I guess it could be considered a bug. You should also check if there are any similar issues first I did find [this](https://github.com/KevinVandy/material-react-table/issues/993) one that seems related although they just said this up to the implementation to deal with the data first like you did so I am not sure they would address anyway.